### PR TITLE
Add details in  quickstart

### LINKED ISSUE #373: [PE-D][Tester D] Quick start section in UG missing some details
In view of your target audience being tutors who may not be tech-savvy, they may not be able to know how to 

1. Check Java version and if it is installed
2. How to open a command terminal and navigate (for each OS)

I believe it will be greatly useful to include some helpful links and sub-pointers that include specific commands to run for greater clarity to the user!

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: fymbc/ped#15

### PR DESCRIPTION
**Pull request by Soumyadip-cmd**
Closes #373

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -20,12 +20,22 @@ If you can type fast, UGTeach can get your contact management tasks done **faste
...
+   * To check the version of java that you have, open a command terminal and run the command `java -version`.
+   * Look for a line that starts with java version. It should display a number, such as 17.0.2. 
+   * Ensure that the version number is 17 or higher.
+   * If Java is not installed, visit the [official Java download page](https://www.oracle.com/java/technologies/downloads/#java17) to download and install the latest version of Java.
+   * After accessing the website, please choose the right download link for your operating system (Linux, macOS or Windows).
 
 2. Download the latest `.jar` file from [here](https://github.com/AY2425S1-CS2103T-F14a-1/tp/releases).
 
 3. Copy the file to the folder you want to use as the _home folder_ for your address book.
+   * By default, the jar file would be in the Downloads folder.
```
Comment (Feng1231) [inline docs/UserGuide.md:32]: Is this only for window users? Or for all users

File: docs/UserGuide.md [hunk 2]
```diff
@@ -20,12 +20,22 @@ If you can type fast, UGTeach can get your contact management tasks done **faste
...
 
 4. Open a command terminal, `cd` into the folder that you put the jar file in.
+   * To open the command terminal, open the terminal app for Linux/macOS users, and Windows
```
Comment (Feng1231) [inline docs/UserGuide.md:35]: Could this be mention at line 23 as well?

**Review by Feng1231 — COMMENTED** [review]
Just some clarifications from me

File: docs/UserGuide.md (hunk 1 above)
Comment (Soumyadip-cmd) [reply docs/UserGuide.md:32]: It seems to be for all os. Can help me confirm @TY1Fan

File: docs/UserGuide.md (hunk 2 above)
Comment (Soumyadip-cmd) [reply docs/UserGuide.md]: I have moved it to line 23

Comment (Feng1231) [reply docs/UserGuide.md]: Okay tq

**Comment by Feng1231** [comment]
Btw do you want to change the title of this PR? quickstart instead of guick

**Review by DennieDan — COMMENTED** [review]
LGTM!

I am not sure if we need to include the `Azul 17` for Mac users, better if we do

**Comment by Soumyadip-cmd** [comment]
> LGTM!
> 
> I am not sure if we need to include the `Azul 17` for Mac users, better if we do

I have added in the step

File: docs/UserGuide.md (hunk 1 above)
Comment (TY1Fan) [reply docs/UserGuide.md]: Yes. For mac when u download it appears in download

**Review by TY1Fan — APPROVED** [review]
LGTM

**Review by teamotitan — APPROVED** [review]
LGTM

**Pull request merged**